src/triage/component/catwalk/feature_importances.py

```diff
@@ -22,7 +23,8 @@ def _ad_hoc_feature_importances(model):
     """
     feature_importances = None
 
-    if isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)):
+    if (isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)) or 
```

Comment:
I guess for statistical reasons is better not to combine feature importances with the coefficients. The SLR it's estimating weights, rather than importances, and the calculation of the latter is mostly only to keep consistency across triage models. We're just rescaling and renaming marginal effects as feature importances here.

Reply:
@thcrock What do you mean ? Like having a feature_importance attribute?

Reply:
@nanounanue yep, having the SLR implement the standard sklearn `.feature_importances_` attribute. You may not want to do it, but maybe you do.

Reply:
Anything new here?